src/components/Slider/Slider.jsx

```diff
@@ -0,0 +1,13 @@
+import './Slider.scss';
+import { useState } from 'react';
+
+export function Slider(props) {
+    const { RangeMin, RangeMax } = props;
+    const { rangeValue } = useState(0);
```

Comment:
Essa não é a implementação correta do useState.

Seria:
`const [ rangeValue, setRangeValue ] = useState(0);`

Acredito que você não precise controlar o estado dentro do componente, e sim aonde ele for utlizado. Acho que o melhor seria receber esse valor por props, e a função que redefina seu valor